README.md

```diff
@@ -17,6 +17,11 @@ DockerCon 2015 Hackathon Project
 
 UI at http://localhost:8008
 
+## Warning
+
+Please be aware, that the default configuration keeps only 2 images: `ubuntu:latest` and `busybox:latest`, so all other images
```

Comment:
Actually that's not completely true. It also does not delete the `rancher/server` image by default : https://github.com/rancherio/sherdock/blob/master/config/config.go#L32